src/main/java/com/algorand/sdkutils/listeners/ResponseGenerator.java

```diff
@@ -370,7 +383,7 @@ public void terminate() {
                         }
                     }
                 });
-        /*
+       
```

Comment:
Could you add some comments to these sections based on our conversations? I realize things aren't perfect, so maybe we can save some time next time with a record of the known issues.